### PR TITLE
update: new proto file, TS fixes

### PR DESCRIPTION
**Pull request by oleksii-donoha**
I've had some problems using the lib with typescript:
 - `EventEmitter` being referenced in the `.d.ts` file, but not being imported
 - nullable `numRequested` fields listed as only `number`, creating TS typing issues
 - etc

Also, the proto file was updated in the SF repository, so I included that.

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -222,12 +224,12 @@ export default class PubSubApiClient {
     /**
      * Subscribes to a topic and retrieves past events starting from a replay ID.
      * @param {string} topicName name of the topic that we're subscribing to
-     * @param {number} numRequested number of events requested. If null, the client keeps the subscription alive forever.
+     * @param {number | null} [numRequested] number of events requested. If null, the client keeps the subscription alive forever.
      * @param {number} replayId replay ID
      * @returns {Promise<EventEmitter>} Promise that holds an emitter that allows you to listen to received events and stream lifecycle events
      * @memberof PubSubApiClient.prototype
      */
-    async subscribeFromReplayId(topicName, numRequested, replayId) {
```
Comment (pozil) [inline src/client.js:230]: Don't change parameter order to ensure backward compatibility.

**Review by pozil — CHANGES_REQUESTED** [review]
Hi @oleksii-donoha and thanks for the contribution, I hadn't noticed the proto file update so this is great.

I don't want to introduce breaking changes so could you apply the requested change?

File: src/client.js (hunk 1 above)
Comment (oleksii-donoha) [reply src/client.js]: done

**Comment by pozil** [comment]
Thanks again @oleksii-donoha. I'm merging the PR and I'll release a new version.

**Pull request merged**